packages/nimble-components/src/table/tests/table.spec.ts

```diff
@@ -195,14 +195,12 @@ describe('Table', () => {
             await element.setData(simpleTableData);
             await waitForUpdatesAsync();
 
-            let headerContent = pageObject.getHeaderContent(0)!.firstChild;
-            expect(headerContent?.textContent).toEqual('stringData');
+            expect(pageObject.getHeaderTextContent(0)).toEqual('stringData');
```

Comment:
If we do add tests for the components page object can we also move it from:
`table/tests/table-pageobject.spec.ts` to
`table/testing/tests/table.pageobject.spec.ts`
to align with file name and test placement patterns a bit better.

Reply:
Based on @jattasNI's guidance, I'm not planning on adding tests to the component's page object. Therefore, I'm not planning on renaming the file as part of this PR.